perfdash/config.go

```diff
@@ -340,6 +340,136 @@ var (
 					Parser:           parsePerfData,
 				}},
 		},
+		"Network_Baseline": {
+			"TCP_P2P": []TestDescription{{
+				Name:             "standalone",
+				OutputFilePrefix: "NetworkPerformanceMetrics1:1TCPP2P",
+				Parser:           parsePerfData,
+			}},
+
+			"TCP_ClusterIP": []TestDescription{
+				{
+					// TODO(oxddr): remove this around Sep '19 when we stop showing old data
```

Comment:
Please remove those TODOs - those are completely unrelated (it was for migration file names, which you're obviously nt doing).

Reply:
It wasn't there in the final commit, something went wrong during splitting(old commit was pushed). Have reverted to the actual changes.